custom-vu/src/main/kotlin/jces1209/vu/page/AttachScreenShot.kt

```diff
@@ -0,0 +1,26 @@
+package jces1209.vu.page
+
+import jces1209.vu.wait
+import org.openqa.selenium.*
+import org.openqa.selenium.interactions.Actions
+import org.openqa.selenium.support.ui.ExpectedConditions
+
+abstract class AttachScreenShot(
```

Comment:
In general there's no reason to make a class `abstract` if it doesn't have any `abstract` members. If you want to open it for inheritance (which should be replaced by composition anyway), then you can use `open class`.